### ISSUE TITLE
[Non-Issue] Additional comma "," in consumes

### ISSUE THREAD

**Issue by wing328** [swagger-api/swagger-codegen#757]
There seems to be an extra "," for all auto-generated code for `consumes` and it seems not causing any issue at the moment (auto-generated codes that I tested are still running fine).

`produces` is fine without the extra ","

Here is a search for `consumes` in the project:
https://github.com/swagger-api/swagger-codegen/search?p=2&q=consumes&utf8=%E2%9C%93

Here is an example for jaxrs [PetApi.java#L29](https://github.com/swagger-api/swagger-codegen/blob/e8ae9178af5a62276f09deac9805ca3ff8b2d6e8/samples/server/petstore/jaxrs/src/main/java/io/swagger/api/PetApi.java#L29):
```
  @Consumes({ "application/json", "application/xml",  })
  @Produces({ "application/json", "application/xml" })
```

**Comment by tomtit** [comment]
Extra commas appeared due to this commit @1385a157e946b1e3354023107e7c6b2b0c354288. The same logic for `produces` left untouched.

**Comment by wing328** [comment]
Addressed by #732 

Pull the latest develop_2.0 and confirmed no more extra comma
```
      # HTTP header 'Accept' (if needed)
      _header_accept = ['application/json', 'application/xml']
      _header_accept_result = Swagger::Request.select_header_accept(_header_accept) and header_params['Accept'] = _header_accept_result

      # HTTP header 'Content-Type'
      _header_content_type = ['application/json', 'application/xml']
```
